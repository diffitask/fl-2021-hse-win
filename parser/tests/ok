Alphabet: 2 -- ("a", "b")
Q: 2
start: 1
runoff: 2
T: 1 -- 2
edges: (1; 2; "a", "b")