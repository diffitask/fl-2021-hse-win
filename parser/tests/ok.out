Alphabet -- 2 symbol(s):
             (a, b)
Start state:
             1
Terminals -- 1 state(s):
             2
All states:
           (number: 1, type: start)
           (number: 2, type: runoff)
Edges:
            1 --> 2, symbols: (a, b)
